src/main/java/net/sf/jabref/gui/JabRefFrame.java

```diff
@@ -162,34 +162,6 @@ public void valueChanged(ListSelectionEvent e) {
         }
     };
 
-    class ToolBar extends OSXCompatibleToolbar {
```

Comment:
Is this really not used on OSX systems? I think there were some OSX classes which were apparently not used but on Mac there were referenced using reflection...